packages/lib-react-components/src/PlainButton/README.md

```diff
@@ -2,4 +2,22 @@
 
 A button component that looks like a plain text link. Has the same API as Grommet's `Button`.
 
-The `Button` label is using the `SpacedText` component. The `text` prop is the string label to be set inside the `SpacedText`. The `SpacedText` size can also be set using the `labelSize` prop, defaults to `medium`. 
+The `Button` label is using the `SpacedText` component. The `text` prop is the string label to be set inside the `SpacedText`. The `SpacedText` size can also be set using the `labelSize` prop, defaults to `medium`.
+
+####Optional prop: color
+The text color can be customized by passing in a string value representing a color.
+
+Example use cases:
+
+```
+<PlainButton
+  color={'teal'}
+/>
+
+or
+
+<PlainButton
+  color={'#008080'}
+/>
+```
```

Comment:
Sorry to be pedantic, but I'd change one of these to illustrate how you can pass in an object. They're both currently showing that a string can be used.

Ex: 
```
{
      dark: "#2d2d2d",
      light: "#eff2f5"
    }
```

Reply:
I agree. Good call.
Updated.